### PR TITLE
Added longest increasing subsequence in Python3 (Issue #820)

### LINKED ISSUE #820: Longest Increasing Subsequence in Python3
This is an open issue for the Longest Increasing Subsequence algorithm implementation in Python3. If you wish to contribute, just make some noise and we will assign you.

### PR DESCRIPTION
**Pull request by mrbrianevans**
**Fixes #820 **

By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md) 

and I am confident that my PR reflects them.

-   [x] I have followed the [coding guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#cs) 
for this project.

-   [x] My code follows the [skeleton code structure](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#sample).

-   [x] This pull request has a descriptive title. 
For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.

-   [x] I have mentioned the issue number correctly (with hyper-link) in this pull request description.
-   [x] This pull request shall only be merged once the build passes.

### CODE REVIEW

File: algorithms/ar-lis/javascript/longest_increasing_subsequence.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/**
+ * Longest increasing subsequence is the subsequence that has the longest streak of going up consecutively
+ * @param {number[]} arr the sequence to search for longest subsequence
+ */
+function findLongestIncreasingSubsequence(arr) {
+    let longestSubsequence = [];
+    for (let arrKey in arr) {
```
Comment (RavinduSachintha) [inline algorithms/ar-lis/javascript/longest_increasing_subsequence.js]: Codacy found an issue: [The body of a for-in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6422328)

File: algorithms/ar-lis/javascript/longest_increasing_subsequence.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        if (Number(arrKey) + 2 > arr.length) break;
```
Comment (RavinduSachintha) [inline algorithms/ar-lis/javascript/longest_increasing_subsequence.js]: Codacy found an issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6422328)

File: algorithms/ar-lis/javascript/longest_increasing_subsequence.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        let tempLength = 0;
+        let isIncreasing = true;
+        while (isIncreasing) { // if next element is greater than current element, subsequence is increasing
+            if (arr[Number(arrKey) + tempLength] >= arr[++tempLength + Number(arrKey)])
```
Comment (RavinduSachintha) [inline algorithms/ar-lis/javascript/longest_increasing_subsequence.js]: Codacy found an issue: [Avoid assignments in operands](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6422328)

File: algorithms/ar-lis/javascript/longest_increasing_subsequence.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+                isIncreasing = false;
+        }
+        // when subsequence is no longer increasing, check if this subsequence is longer than any other previous one
+        if (tempLength > longestSubsequence.length)
```
Comment (RavinduSachintha) [inline algorithms/ar-lis/javascript/longest_increasing_subsequence.js]: Codacy found an issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6422328)

File: algorithms/ar-lis/javascript/longest_increasing_subsequence.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    }
+    return longestSubsequence;
+}
+
+function main() {
+    let input = [
+        3, 1, 4, 1, 5, 9, 2, 6, 5, 3, 5, 8, 9, 7, 9, 3, 2, 3, 8, 4, 6, 2, 6,
+        4, 3, 3, 8, 3, 2, 7, 9, 5, 0, 2, 8, 8, 4, 1, 9, 7, 1, 6, 9, 3, 9, 9
+    ];
+    console.log(findLongestIncreasingSubsequence(input));
```
Comment (RavinduSachintha) [inline algorithms/ar-lis/javascript/longest_increasing_subsequence.js]: Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6422328)

File: algorithms/ar-lis/javascript/longest_increasing_subsequence.js (hunk 3 above)
Comment (RavinduSachintha) [inline algorithms/ar-lis/javascript/longest_increasing_subsequence.js]: Codacy found an issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6422328)

**Comment by Danushka96** [comment]
Hi @mrbrianevans 
Can you update the main Readme.md file algorithm chart with the algorithm you implemented?

**Comment by mrbrianevans** [comment]
> Can you update the main Readme.md file algorithm chart with the algorithm you implemented?

Okay, I have added it

**Review by Danushka96 — APPROVED** [review]

**Comment by Danushka96** [comment]
@all-contributors please add @mrbrianevans for code and doc

**Pull request merged**